docs/lang/articles/basic/syntax.md

```diff
@@ -54,21 +55,21 @@ Kernel arguments must be type-hinted:
 def my_kernel(x: ti.i32, y: ti.f32):
     print(x + y)
 
-my_kernel(2, 3.3)  # prints: 5.3
+my_kernel(24, 3.2)  # prints: 27.2
 ```
 
 :::note
 
-For now, we only support scalars as arguments. Specifying `ti.Matrix` or
-`ti.Vector` as argument is not supported. For example:
+For now, Taichi supports scalars as kernel arguments. Specifying `ti.Matrix` or
+`ti.Vector` as an argument is not supported yet:
 
 ```python {2,6}
 @ti.kernel
-def bad_kernel(v: ti.Vector):
+def error_kernel(v: ti.Vector): # Error
```

Comment:
Could we exchange the order of `error_kernel` and `valid_kernel`? Because you said `scalars as kernel arguments` first.